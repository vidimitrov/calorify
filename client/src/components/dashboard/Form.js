import styled from 'styled-components';

const Form = styled.div`
  padding: 32px;
`;

export default Form;
