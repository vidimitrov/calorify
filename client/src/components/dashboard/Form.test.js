import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Form from './Form';

it('Dashboard/Form component renders correctly', () => {
  const tree = renderer.create(<Form />).toJSON();
  expect(tree).toMatchSnapshot();
});
