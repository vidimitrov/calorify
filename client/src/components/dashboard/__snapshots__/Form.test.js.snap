// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/Form component renders correctly 1`] = `
.c0 {
  padding: 32px;
}

<div
  className="c0"
/>
`;
